modules/ROOT/pages/deriving-container.adoc

```diff
@@ -0,0 +1,94 @@
+= Deriving from Fedora CoreOS as a bootable container image
+
+== Bootable containers overview
+
+The operating system images used for Fedora CoreOS are available as a standard container image; there is one per stream:
+
+- quay.io/fedora/fedora-coreos:stable
+- quay.io/fedora/fedora-coreos:next
+
+etc.
+
+== Creating custom builds
+
+See the https://github.com/coreos/layering-examples[layering examples] git repository.  While the examples there tend to use Container/Dockerfile style builds, you can use any tool which can build containers and push the result to a registry.
+
+== Using Ignition to retarget an instance
+
+The https://coreos.github.io/rpm-ostree/container/[rpm-ostree container page] describes the commands for interacting with bootable ostree container images.
+
+In this example, we will use a systemd unit to fetch and reboot into the target custom container image, and also install a systemd unit which will enable
+automatic polling of the target tag, and reboot on changes.
+
+[source,yaml]
+----
+variant: fcos
+version: 1.4.0
+systemd:
+  units:
+    # Our custom unit for rebasing
+    - name: rebase-custom.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Fetch and deploy target image
+        # Only run on the firstboot
+        ConditionFirstBoot=true
+        After=network-online.target
+        [Service]
+        # This ordering is important
+        After=ignition-firstboot-complete.service
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStart=rpm-ostree rebase --reboot ostree-unverified-registry:quay.io/example/example-derived:latest
+        [Install]
+        WantedBy=multi-user.target
+    # Our custom unit for automatic upgrades
+    - name: autoupdate-host.service
+      enabled: false
+      contents: |
+        [Unit]
+        Description=Automatic host upgrades
+        [Service]
+        Type=simple
+        ExecStart=rpm-ostree upgrade --reboot
+        StandardOutput=null
```

Comment:
`StandardOutput=null` seems like an odd thing to do. Why?

Reply:
Because otherwise you get many log messages to the journal twice (once from the client, once from the daemon).

Reply:
Maybe a comment here would be useful 
```
# Prevent duplicate logging from daemon and this unit
```